beamer/util.py

```diff
@@ -67,7 +67,7 @@ def transact(
     receipt = func.web3.eth.wait_for_transaction_receipt(
         txn_hash, timeout=timeout, poll_latency=poll_latency
     )
-    if receipt.status == 0:
+    if receipt["status"] == 0:
```

Comment:
This change should be squashed into the first commit of the PR, it does not belong in this commit (which is about the test).